src/classes/GE_GiftEntryController.cls

```diff
@@ -40,6 +40,7 @@ public with sharing class GE_GiftEntryController {
     public static final String PURCHASE_CALL_TIMEOUT_MESSAGE = System.Label.geErrorRequestTimedout;
     private static final String BOOLEAN_DATA_TYPE = 'BOOLEAN';
     private static GiftBatchService giftBatchService = new GiftBatchService();
+    private static GiftService giftService = new GiftService();
```

Comment:
This is the pattern we are currently following to allow for injection:
![2021-07-09_13-39-05](https://user-images.githubusercontent.com/46458081/125133712-1de53980-e0bb-11eb-8d4f-fc02da81dca0.png)